include/service_control_client.h

```diff
@@ -281,6 +281,17 @@ class ServiceControlClient {
       DoneCallback on_check_done, TransportCheckFunc check_transport) = 0;
 
   // An async quota call.
+  // on_quota_done is called with the quota request status after cached
+  // quota_response is returned in case of cache hit, otherwise called after
+  // quota_response is returned from the Controller service.
```

Comment:
> otherwise called after `quota_response` is returned from the controller service

this is not true, we do not block for controller service. we return a dummy response

Reply:
Yes indeed. I copied this from `Check()` comments because I mistakenly thought they would be the same. Updated the comments.